ci/scripts/jenkins_build.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+source ci/scripts/jenkins_common.sh
+
+gpuci_logger "Creating conda env"
+conda config --add pkgs_dirs /opt/conda/pkgs
+conda config --env --add channels conda-forge
+conda config --env --set channel_alias ${CONDA_CHANNEL_ALIAS:-"https://conda.anaconda.org"}
+mamba install -q -y -n base -c conda-forge "boa >=0.10"
+mamba create -q -y -n morpheus python=${PYTHON_VER}
+conda activate morpheus
+
+gpuci_logger "Installing CI dependencies"
+mamba env update -q -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_ci.yml
+
+gpuci_logger "Check versions"
+python3 --version
+gcc --version
+g++ --version
+
+gpuci_logger "Check conda environment"
+conda info
+conda config --show-sources
+conda list --show-channel-urls
+
+gpuci_logger "Checking S3 cuDF cache"
+CONDA_BLD_DIR=/opt/conda/conda-bld
```

Comment:
This variable name is very similar to `CONDA_BLD_PATH` which has very similar but different functionality. Can we change this name so it doesnt get mixed up? Docs are here: https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#specify-conda-build-output-root-directory-root-dir